### PR TITLE
feat(extract hdr): Add component using pinterp for HDR conversion

### PR DESCRIPTION
**Pull request by mikkelkp**
Should only be merged after this: https://github.com/ladybug-tools/honeybee-radiance-command/pull/204

### CODE REVIEW

File: honeybee_grasshopper_radiance/src/HB Extract HDR.py
Comment (chriswmackey) [inline honeybee_grasshopper_radiance/src/HB Extract HDR.py:79]: Do we need this variable? Why not just set `hdr_view_ to None here and then later check if its still None.

File: honeybee_grasshopper_radiance/src/HB Extract HDR.py
Comment (chriswmackey) [inline honeybee_grasshopper_radiance/src/HB Extract HDR.py:92]: We don't need these two checks here if you are just going to check the view type on the object later.

File: honeybee_grasshopper_radiance/src/HB Extract HDR.py
Comment (chriswmackey) [inline honeybee_grasshopper_radiance/src/HB Extract HDR.py:28]: I usually try to put the required inputs towards the top of the component. It seems a little unconventional to have one at the bottom that isn't a boolean toggle for `_run`.

File: honeybee_grasshopper_radiance/src/HB Extract HDR.py
Comment (chriswmackey) [inline honeybee_grasshopper_radiance/src/HB Extract HDR.py:157]: This is not PEP8. We need a line break here.

File: honeybee_grasshopper_radiance/src/HB Extract HDR.py
Comment (chriswmackey) [inline honeybee_grasshopper_radiance/src/HB Extract HDR.py:149]: Also not PEP8.

File: honeybee_grasshopper_radiance/src/HB Extract HDR.py
Comment (chriswmackey) [inline honeybee_grasshopper_radiance/src/HB Extract HDR.py:163]: PEP8 says you should put spaces around operators like ` / `.

File: honeybee_grasshopper_radiance/src/HB Extract HDR.py
Comment (chriswmackey) [inline honeybee_grasshopper_radiance/src/HB Extract HDR.py:124]: Am I correct that the Radiance command still succeeds even when the output resolution is larger than the input resolution? If so, this is good. Otherwise, it would be good to raise an error here instead of a warning.

**Review by chriswmackey — CHANGES_REQUESTED** [review]
Thanks @mikkelkp .

It's generally looking good and I just have a few comments that should be addressed before we merge.

File: honeybee_grasshopper_radiance/src/HB Extract HDR.py
Comment (mikkelkp) [reply honeybee_grasshopper_radiance/src/HB Extract HDR.py]: I don't mind. I just put it under '3' because the other HDR postprocessing components are there.

File: honeybee_grasshopper_radiance/src/HB Extract HDR.py
Comment (mikkelkp) [reply honeybee_grasshopper_radiance/src/HB Extract HDR.py:119]: Yes, that is correct.

**Comment by mikkelkp** [comment]
Thanks @chriswmackey! I have addressed your requested changes.

**Review by chriswmackey — APPROVED** [review]
Looks good, @mikkelkp . Thank you. I'll merge it once your honeybee-radiance-command update makes it through the CI. Then, I'll add an icon for it.

**Pull request merged**
